backend/src/main/java/de/silvia/backend/models/ArtikelList.java

```diff
@@ -0,0 +1,30 @@
+package de.silvia.backend.models;
+
+import lombok.*;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+import java.util.List;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Document("value(listname)")
```

Comment:
Was versuchst du hier? Da gibst du eigentlich nur den Namen der Collection in der DB an. z.B. "ArtikelLists"